python/contextscene/ContextScene.py

```diff
@@ -0,0 +1,370 @@
+# Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+# See LICENSE.md in the project root for license terms and full copyright notice.
+
+import os
+import json
+
+'''
+Python structure for basic context scene creation
```

Comment:
It would be a nice addition to have docstrings in classes and functions so that customers have a baseline on how to use the code in completent to the examples.